### ISSUE TITLE
NVDA Doesn't Read Incoming Teams Messages

### ISSUE THREAD

**Issue by munawarb** [nvaccess/nvda#13799]
### Steps to reproduce:
1. Open Teams
2. Focus on a chat input field.
3. Wait for the other party to send a message.
4. Notice that NVDA no longer announces the incoming message and instead announces "Sending...".

### Actual behavior:
NVDA doesn't announce incoming messages.

### Expected behavior:
NVDA should announce incoming and outgoing messages. Right now, if you're focused on a chat window, the only way to tell if the other party has responded is by periodically checking the chat history list.

**Comment by seanbudd** [comment]
Have you reported this to Microsoft? 
These pages may be helpful 

- [Accessibility feedback and help](https://www.microsoft.com/en-us/accessibility/feedback).
- [How to give feedback in MS Teams](https://support.microsoft.com/en-us/office/give-feedback-in-teams-c0fb6297-22af-4db5-b19b-69e0a6720927).

**Comment by munawarb** [comment]
Yes I did, also told them about #13081.

**Comment by rperez030** [comment]
Hi @munawarb . If you are an enterprise customer, which you probably are if you are using Teams, I suggest you write an email to edad@microsoft.com telling them about this issue and how having incoming messages read automatically by NVDA is important for you. In my testing this is not working with any of the Windows screen readers at the moment.

**Comment by Adriani90** [comment]
@munawarb do you still have this issue  with the last Teams Update and the newest NVDA version?

**Comment by munawarb** [comment]
@Adriani90 This is no longer an issue. Looks like Microsoft has fixed Teams.

**Comment by seanbudd** [comment]
Thanks, closing
